### PR TITLE
activity tracking article

### PR DESCRIPTION
**Pull request by meeunier**
As per #250 , this PR includes the draft for the activity tracking article inspired from the [blog post](https://blog.dnsimple.com/2016/12/announcing-activity-tracking/).

### CODE REVIEW

File: content/articles/activity-tracking.markdown [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: Activity Tracking
+excerpt: How to manage the users on a DNSimple account.
```
Comment (weppos) [inline content/articles/activity-tracking.markdown]: I think the excerpt doesn't reflect this article.

File: content/articles/activity-tracking.markdown [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+categories:
+- Account
+---
+
+# Managing account users
```
Comment (weppos) [inline content/articles/activity-tracking.markdown]: I think the title doesn't reflect this article.

File: content/articles/activity-tracking.markdown [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+<note>
+Personal plan subscribers have access to all events that occurred during the last 30 days. A [Professional and Business subscription](https://dnsimple.com/pricing) is required to view the full history.
```
Comment (weppos) [inline content/articles/activity-tracking.markdown]: We generally link the /new-plans#newer-plans-some in this case.
See e.g. https://github.com/dnsimple/dnsimple-support/blob/master/content/articles/regional-records.markdown

I'd also rephrase

> is required to view the full history.

as it sees mandatory. I'd use something like:

> Audit history is available only for new plans. Full audit history is available for Professional and Business plans, Personal plans audit history is limited to 30 days.

File: content/articles/activity-tracking.markdown [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+</note>
+
+Activity tracking provides an overview of all past changes for a specific domain or for an account as a whole.
+
+## Activity tracking at the domain level
+
+Domain level activity tracking can be found under the domain page.
+
+![Activity Tracking at domain level](/files/activity-tracking-1.png)
+![Activity Tracking at domain level](/files/activity-tracking-2.png)
```
Comment (weppos) [inline content/articles/activity-tracking.markdown]: I think this screenshot is old. It still contains the arrows.

File: content/articles/activity-tracking.markdown [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+<note>
+Personal plan subscribers have access to all events that occurred during the last 30 days. A [Professional and Business subscription](https://dnsimple.com/pricing) is required to view the full history.
+</note>
+
+Activity tracking provides an overview of all past changes for a specific domain or for an account as a whole.
+
+## Activity tracking at the domain level
+
+Domain level activity tracking can be found under the domain page.
```
Comment (weppos) [inline content/articles/activity-tracking.markdown]: In screenshot 1 I'd show a larger section, as you did for the account, to help understand where it is located. The "domain page" is vague, we should indicate in which tab and where precisely.

Having a screenshot that shows its position would help as well.

File: content/articles/activity-tracking.markdown [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+Account level activity tracking can be found under the account page.
+
+![Activity Tracking at account level](/files/activity-tracking-3.png)
+![Activity Tracking at account level](/files/activity-tracking-4.png)
+
+## Events filter
+
+<note>
+A [Professional and Business subscription](https://dnsimple.com/pricing) is required to use event filters.
```
Comment (weppos) [inline content/articles/activity-tracking.markdown]: As before, I'd rephrase:

> Full audit history is available only on certain plans.

**Comment by weppos** [comment]
Thanks for writing the article. I left some comments.

I also wonder if we should rename this feature from activity to audit. It seems a much better (and more appropriate) naming, as this is often referred as audit log.

File: content/articles/activity-tracking.markdown (hunk 1 above)
Comment (meeunier) [inline content/articles/activity-tracking.markdown]: Indeed #caughtcopypasting. Fixed it.

File: content/articles/activity-tracking.markdown (hunk 2 above)
Comment (meeunier) [inline content/articles/activity-tracking.markdown]: #caughtcopypasting, again. Fixed it.

File: content/articles/activity-tracking.markdown (hunk 3 above)
Comment (meeunier) [inline content/articles/activity-tracking.markdown]: 👍

File: content/articles/activity-tracking.markdown (hunk 5 above)
Comment (meeunier) [inline content/articles/activity-tracking.markdown]: 👍

**Comment by meeunier** [comment]
@weppos I've added your suggestions.

As for "Audi log" vs "Activity Tracking" ----> 🤷‍♂️

**Comment by meeunier** [comment]
😳 uh oh, didn't realized @arnamak hadn't gave the 👍 yet. 

Sorry pal. Let me know if you want to tweak it
